package.json

```diff
@@ -59,6 +59,7 @@
     "@jupyterlab/observables": "^3.0.0-beta.2",
     "@jupyterlab/notebook": "^2.0.0-beta.2",
     "@jupyterlab/services": "^5.0.0-beta.2",
+    "@jupyterlab/ui-components": "^2.0.0-beta.2",
```

Comment:
Keeping `beta.2` here for now. The full upgrade to final can be done all at once.